api/nakadi-event-bus-api.yaml

```diff
@@ -548,6 +548,43 @@ paths:
           description: |
             Schema object.
 
+  '/event-types/{name}/cursor-distances':
+    get:
+      tags:
+        - unmanaged-api
+        - monitoring
+        - management-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Calculate the distance between two offsets. This is useful for performing checks for data completeness, when
+        a client has read some batches and wants to be sure that all delivered events have been correctly processed.
+      parameters:
+        - name: X-nakadi-cursor-distances-query
+          in: header
+          description: |
+            List of cursors with two special offsets: `initial_offset` and `final_offset`. Used as input by Nakadi to
+            calculate how many events there are between two offsets. The distance doesn't include the `initial_offset`
+            but includes the `final_offset`. So they are equal the result is zero. If the `initial_offset` is the one
+            before the last, the result is 1.
+          type: array
+          items:
+            type: string
+            format: "#/definitions/CursorDistanceQuery"
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            description: An array of `CursorDistanceResult`s with the distance between two offsets.
+            items:
+              $ref: '#/definitions/CursorDistanceResult'
+        '401':
```

Comment:
you can skip "default" status codes in swagger file until they have some special meaning/use-case